.travis.yml

```diff
@@ -1,62 +1,54 @@
 language: python
 python:
-  - 2.7_with_system_site_packages
-  - 3.5.1
-  - 3.6.2
-  - pypy
+  - 2.7
+  - 3.5
+  - 3.6
 cache:
+  pip: true
+  apt: true
   directories:
-    - $HOME/.cache/pip
+    - /tmp/ffmpeg/bin
 sudo: false
+dist: trusty
 matrix:
   fast_finish: true
   allow_failures:
-    - python: '3.5.1'
-    - python: '3.6.2'
-    - python: pypy
+    - python: 3.5
+    - python: 3.6
 env:
   - INTEGRATION_TEST=1
   - UNIT_TEST=1
 addons:
   apt:
     packages:
-    - libcurl4-openssl-dev
-    - python-numpy
-    - python-opencv
-    - libimage-exiftool-perl
-    - libopencv-dev
     - libjpeg-progs
-    - libjpeg-dev
-    - libpng-dev
+    - libimage-exiftool-perl
+    - gifsicle
+    - python-all-dev
+    - scons
+    - libboost-python-dev
+    - libexiv2-dev
     - libx264-dev
     - libass-dev
-    - libvpx1
     - libvpx-dev
-    - libwebp-dev
-    - webp
-    - gifsicle
-    - python-numpy
-    - python-scipy
-    - libcairo2-dev
-    - gfortran
-    - libopenblas-dev
-    - liblapack-dev
-    - python-pyexiv2
     - yasm
 install:
   - pip install -U --upgrade pip
```

Comment:
Could this be `--user` instead of `-U`?  Explicit is better than...  I forgot to do this on an earlier PR.